alpha/lib/model/entry.php

```diff
@@ -3611,7 +3611,11 @@ private function getAllUserNamesAsArray()
 		}
 
 		if($result)
+		{
+			$result = array_unique($result);
```

Comment:
look like the order of these calls should be replaced , in case that same value is used for some of field , no point in taking it together.